### ISSUE TITLE
pulls.lua:20 script ran too long

### ISSUE THREAD

**Issue by jjholleman** [Nnoggie/MythicDungeonTools#235]
When opening MDT, either through `/mdt`, minimap or titan panel icon: the game freezes and after ~10 seconds resumes with this LUA error.

This usually indicates an infinite loop.

All addons were turned off for this test, except for MDT, Bugsack and Buggrabber.
MDT version: `v3.5.5.`

Full LUA error:
```lua
1x MythicDungeonTools\pulls.lua:20: script ran too long
[string "@MythicDungeonTools\pulls.lua"]:20: in function <MythicDungeonTools\pulls.lua:16>
[string "@MythicDungeonTools\pulls.lua"]:36: in function <MythicDungeonTools\pulls.lua:24>
[string "@MythicDungeonTools\pulls.lua"]:161: in function `DrawHull'
[string "@MythicDungeonTools\pulls.lua"]:203: in function `DrawAllHulls'
[string "@MythicDungeonTools\DungeonEnemies.lua"]:898: in function `DungeonEnemies_UpdateSelected'
[string "@MythicDungeonTools\MythicDungeonTools-3.5.5.lua"]:3905: in function `SetSelectionToPull'
[string "@MythicDungeonTools\MythicDungeonTools-3.5.5.lua"]:2978: in function `UpdateMap'
[string "@MythicDungeonTools\MythicDungeonTools-3.5.5.lua"]:1600: in function `SetAffixWeek'
[string "@MythicDungeonTools\MythicDungeonTools-3.5.5.lua"]:5189: in function <...aceMythicDungeonTools\MythicDungeonTools.lua:4970>
[string "@MythicDungeonTools\MythicDungeonTools-3.5.5.lua"]:662: in function `ShowInterface'
[string "@MythicDungeonTools\MythicDungeonTools-3.5.5.lua"]:62: in function `?'
[string "@FrameXML\ChatFrame.lua"]:5094: in function `ChatEdit_ParseText'
[string "@FrameXML\ChatFrame.lua"]:4758: in function `ChatEdit_SendText'
[string "@FrameXML\ChatFrame.lua"]:4794: in function `ChatEdit_OnEnterPressed'
[string "*:OnEnterPressed"]:1: in function <[string "*:OnEnterPressed"]:1>

Locals:
a = <table> {
 1 = 340.706490
 2 = -195.924454
 3 = 5.460000
}
b = <table> {
 1 = 325.695209
 2 = -189.856276
 3 = 5.460000
}
c = <table> {
 1 = 325.695209
 2 = -198.393876
 3 = 5.460000
}
u1 = -15.011282
v1 = 6.068177
u2 = -15.011282
(*temporary) = -198.393876
(*temporary) = -195.924454
(*temporary) = nil
(*temporary) = "script ran too long"

```

**Comment by jjholleman** [comment]
My SavedVariables file if that helps
[MythicDungeonTools.zip](https://github.com/Nnoggie/MythicDungeonTools/files/6291758/MythicDungeonTools.zip)

**Comment by Blacknblue123** [comment]
I also have this problem. One thing I've noticed is when you log on you character and click the button it freezes BUT when you enter any dungeon on any difficulty it starts working fine even when you exit a dungeon.

**Comment by Nnoggie** [comment]
fixed in v3.5.6
